herd/unittests/AArch64/A128.litmus

```diff
@@ -0,0 +1,9 @@
+AArch64 A128
+(* Tests accessing a location - symbolic location *)
+{  0:X2=v; v[1] = 44; v[0] = 22;}
```

Comment:
This init section puzzles me. I rather see init section as _global_ declarations (well up to register initialisations..) Hence writing `v[1] = 44` without allocating the space for array `v` looks disturbing to me.

From the  perspective of the litmus tool, the idea of reserving space in the init section may be more apparent. Writing `int x` tells litmus to reserve space for some int, writing `uint64_t x` to reserve 8-byte etc.